toffy/streak_detection.py

```diff
@@ -185,47 +185,53 @@ def _make_mask_dataframe(streak_data: StreakData, min_length: int = 50) -> None:
     Args:
         streak_data (StreakData): An instance of the StreakData Dataclass, holds all necessary
         data for streak correction.
-        min_length (int): The lower threshold for filtering streaks in pixels. Defaults to 50.
+        min_length (int): The lower threshold for filtering streaks in pixels. Defaults to 70.
     """
     # Label all the candidate streaks
     labeled_streaks = measure.label(streak_data.streak_mask, connectivity=2, return_num=False)
 
-    # Gather properties of all the candidate streaks using regionprops.
-    region_properties = measure.regionprops_table(
-        label_image=labeled_streaks,
-        cache=True,
-        properties=[
-            "label",
-            "bbox",
-            "eccentricity",
-            "area",
-        ],
-    )
+    # if streaks detected, filter dataframe
```

Comment:
This crashes if there are no streaks to filter, so I added a check before running the filtering code